apps/src/code-studio/components/progress/ScriptOverview.jsx

```diff
@@ -45,6 +45,18 @@ class ScriptOverview extends React.Component {
     showAssignButton: PropTypes.bool,
     assignedSectionId: PropTypes.number,
     minimal: PropTypes.bool,
+    lessons: PropTypes.arrayOf(
+      PropTypes.shape({
```

Comment:
It would be great to use the new unitCalendarLesson shape here

Reply:
Also I think these lessons are specifically for the unit calendar right? So they should probably be called out that way maybe unitCalendarLessons